test/testBtc.ts

```diff
@@ -0,0 +1,327 @@
+// Tests for BTC transaction edge cases
+// NOTE: You must run the following BEFORE executing these tests:
+//
+// 1. Pair with the device once. This will ask you for your deviceID, which will
+//    act as a salt for your pairing:
+//
+//    env REUSE_KEY=1 npm run test
+//
+// 2. Connect with the same deviceID you specfied in 1:
+//
+//    env DEVICE_ID='<your_device_id>' npm test
+//
+// After you do the above, you can run this test with `env DEVICE_ID='<your_device_id>' npm run test-btc`
+//
+// NOTE: It is highly suggested that you set `AUTO_SIGN_DEV_ONLY=1` in the firmware
+//        root CMakeLists.txt file (for dev units)
+require('it-each')({ testPerIteration: true });
+// Bitcoin specific functionality/setup
+// !!!! IMPORTANT NOTE: YOU MUST RUN THESE TESTS AGAINST A LATTICE WHOSE SEED WAS RECOVERED USING
+//                      THE MNEMONIC SPECIFIED BELOW!!!!
+// Set up the wallet:
+import bip32 from 'bip32';
+import { expect as expect } from 'chai';
+import seedrandom from 'seedrandom';
+import helpers from './testUtil/helpers';
+const prng = new seedrandom(process.env.SEED || 'myrandomseed');
+const TEST_TESTNET = !!process.env.TESTNET || false;
```

Comment:
Unrelated to this PR but for your upcoming docs let's make sure we enumerate all possible flags (and default values) for each test